fili-core/src/main/java/com/yahoo/bard/webservice/web/util/QuerySignedCacheService.java

```diff
@@ -107,14 +112,23 @@ public String readCache(
                     }
                     CACHE_HITS.mark(1);
                     BardQueryInfo.getBardQueryInfo().incrementCountCacheHits();
+                    BardQueryInfo.getBardQueryInfo().addCacheReadInfo(
+                            CacheV2ResponseProcessor.getMD5Cksum(getKey(druidQuery)),
```

Comment:
when we're doing a CacheV2 Response processor, we should capture a checksum of the cache signature as well.